libs/providers/flagd/src/lib/flagd-provider.spec.ts

```diff
@@ -1,11 +1,12 @@
 jest.mock('@protobuf-ts/grpc-transport');
 
-import { Client, ErrorCode, EvaluationContext, OpenFeature } from '@openfeature/js-sdk';
+import { Client, ErrorCode, EvaluationContext, JsonObject, OpenFeature, StandardResolutionReasons } from '@openfeature/js-sdk';
```

Comment:
Can you make sure this is formatted with Prettier?

Reply:
I will check that.